stardog_setter.py

```diff
@@ -36,17 +37,32 @@ def sub_bad_chars(data):
     else:
         return data
 
-KINDS = ['namespaces', 'pods', 'deployments']
+
+KINDS = ["namespaces", "pods", "deployments"]
+DONEFILE = "dirsdone.json"
+
+
 def process(stardog_data_dir, astra_proxy_url):
+    done = []
+    if path.exists(DONEFILE):
```

Comment:
If you want to be gross and pythonic, you could also try/except on the relevant file not found error and `pass`...honestly idk why it's done that way in the python world